BraveShared/BraveStrings.swift

```diff
@@ -512,6 +512,8 @@ extension Strings {
     public static let HTTPSEverywhere = NSLocalizedString("HTTPSEverywhere", tableName: "BraveShared", bundle: Bundle.braveShared, value: "HTTPS Everywhere", comment: "")
     public static let HTTPSEverywhereDescription = NSLocalizedString("HTTPSEverywhereDescription", tableName: "BraveShared", bundle: Bundle.braveShared, value: "Opens some sites using a secure HTTPS connection instead of plain HTTP.", comment: "")
     public static let blockPhishingAndMalware = NSLocalizedString("BlockPhishingAndMalware", tableName: "BraveShared", bundle: Bundle.braveShared, value: "Block Phishing and Malware", comment: "")
+    public static let googleSafeBrowsing = NSLocalizedString("GoogleSafBrowsing", tableName: "BraveShared", bundle: Bundle.braveShared, value: "Block dangerous sites", comment: "")
+    public static let googleSafeBrowsingDescription = NSLocalizedString("GoogleSafBrowsingDescription", tableName: "BraveShared", bundle: Bundle.braveShared, value: "Sends obfuscated URLs of some pages you visit to Brave to check against the Google Safe Browsing service, when your security is at risk.", comment: "")
```

Comment:
on iOS 14 the obfuscated URLs are not sent to Brave - 

```
Sends obfuscated URLs of some pages you visit to the Google Safe Browsing service, when your security is at risk.
```

cc: @fmarier

Reply:
I agree with @jumde based on the description on https://support.apple.com/en-us/HT210675.